Makefile

```diff
@@ -200,3 +182,21 @@ $(NODE_MODULES_TEST): package.json
 	env -u NODE_ENV npm prune
 
 .PHONY: all clean install devel-install dist node-cache rpm check vm update-po
+
+# checkout common files from Cockpit repository required to build this project;
+# this has no API stability guarantee, so check out a stable tag when you start
+# a new project, use the latest release, and update it from time to time
+COCKPIT_REPO_FILES = \
+	pkg/lib \
+	test/common \
+	$(NULL)
+
+COCKPIT_REPO_URL = https://github.com/cockpit-project/cockpit.git
+COCKPIT_REPO_SHA = d1a410cf9ccb33e983fbd38cad5a6fc95bc1c4b4 # 264-137-gd1a410cf9
+
+COCKPIT_REPO_STAMP = pkg/lib/cockpit-po-plugin.js
```

Comment:
See above, let's move this up and re-use in the webpack rule.

Reply:
ya ya ya